src/metabase/driver/common/parameters/values.clj

```diff
@@ -24,6 +26,7 @@
 (def ^:private ParamType
   (s/enum :number
           :dimension                    ; Field Filter
+          :question
```

Comment:
while consistent with our external nomenclature, we use card elsewhere in the code. Not sure if it's a good idea to use both terms interchangeably.

Reply:
Good catch. I'm unsure about where the boundary is where a "question" becomes a "card", so I opted for the front-end-friendliest option.

From your comment it seems like that boundary is where the front-end generates queries: before that it's a "question", but in the query it's a "car". Is that correct?

Reply:
I think throughout the BE it's card (even the endpoint is /card)

Reply:
> even the endpoint is /card

That's only really because we haven't taken the time to update the API.

I guess it's not that big of a deal, but I don't think we shouldn't be digging ourselves deeper by adding `card` to the API surface area, which we'll probably eventually want to migrate to `question`. I believe the only places we use `card` in the API are the `/api/card` endpoints (which could/should trivially be renamed to `/api/question` IMHO), and in dashboards.

We do use `question` in JWT embedding, for example(but that is explicitly a user-facing API)